scala3doc/src/main/scala/dotty/dokka/tasty/BasicSupport.scala

```diff
@@ -0,0 +1,171 @@
+package dotty.dokka.tasty
+
+import org.jetbrains.dokka.links._
+import org.jetbrains.dokka.model._
+import collection.JavaConverters._
+import dotty.dokka._
+import dotty.dokka.model.api.Visibility
+import dotty.dokka.model.api.VisibilityScope
+import dotty.dokka.model.api.Modifier
+import dotty.dokka.model.api.Annotation
+
+
+import scala.tasty.Reflection
+
+trait BasicSupport:
+  self: TastyParser =>
+  import reflect._
+
+  object SymOps extends SymOps[reflect.type](reflect)
```

Comment:
It looks like this is encoding what is described in #9530. We should improve extensions to support this use case.

Reply:
Yeah, I was not happy when I needed to introduce Cake Patter that cause me so much pain in Scala 2